### ISSUE TITLE
Prettier formatting not working

### ISSUE THREAD

**Issue by rushikeshRL** [getcursor/cursor#1208]
1) If you can, please include a screenshot of your problem
<img width="285" alt="image" src="https://github.com/getcursor/cursor/assets/115528161/52c81ae8-6f19-447e-a6ae-76b0425054c4">

2) Please include the name of your operating system
Windows 11
3) If you can, steps to reproduce are super helpful
Download prettier extension. Open ts or tsx file. Try to format using format document command.

**Comment by mishschmid** [comment]
Same here, also for other file types like `.vue` files. 

`Prettier` overall doesn't seem to work in Cursor, also checked out other repos. The exact same settings work fine in VS Code.

**Comment by chrislodev** [comment]
Yep I've also just found this. Prettier loads and works in VS Code. In Cursor I get a "Cannot find Prettier package.json" error, no matter if I try to use the project's node module or a globally installed one.

**Comment by seanroades** [comment]
Any fixes?

**Comment by miracim** [comment]
Yes, it worked for me. Prettier - I downgraded the version of the Code Formatter extension from 11.0.0 to 10.4.0. This worked for me

**Comment by natec8** [comment]
I got it to work by installing prettier globally using yarn: `yarn global add prettier`. Then I added this to the the prettier settings as the Prettier Path (I'm on a mac)

`/Users/<user>/.config/yarn/global/node_modules/prettier/index.cjs`

**Comment by AzizFacilex** [comment]
Same here. I had to downgrade prettier to 10.4.0 to work. The 11.0.0 is working in VSCode though...
